nixos/modules/services/networking/unbound.nix

```
@@ -102,8 +102,8 @@ in {
           freeformType = let
             validSettingsPrimitiveTypes = oneOf [ int str bool float ];
             validSettingsTypes = oneOf [ validSettingsPrimitiveTypes (listOf validSettingsPrimitiveTypes) ];
-            settingsType = (attrsOf validSettingsTypes);
-          in attrsOf (oneOf [ string settingsType (listOf settingsType) ])
+            settingsType = oneOf [ string (attrsOf validSettingsTypes) ];
```

```suggestion
            settingsType = oneOf [ str (attrsOf validSettingsTypes) ];
```

`string` is a deprecated type.

---

The intermediate types could also use a human readable description to avoid gollum showing up in error messages:
```
error: A definition for option `services.unbound.settings.include.[definition 1-entry 1]' is not of type `attribute set of signed integer or string or boolean or floating point number or list of signed integer or string or boolean or floating point numberss'.
```

---

Fixed the deprecated `string` type. Sorry about that.

---

Don't worry about it. We don't have a usable warning for it. #99132 broke other things.